build/workbox.config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "globDirectory": "./",
+    "globPatterns": [
+        "assets/**/*.{js,svg,png,css}",
+        "dist/**/*.{js,css}",
+        "docs/**/*.{md,html,css,jpg,js}"
```

Comment:
Also, this seems to generic. You should change the whole config to work against the _gh_pages folder which only has the needed assets and not the src stuff.